invenio_github/api.py

```diff
@@ -331,31 +321,29 @@ def repository(self):
         """Return repository metadata."""
         return self.payload['repository']
 
-    @property
-    def repository_model(self):
+    @cached_property
```

Comment:
This might be dangerous because you can get an object bounded to a different session or transaction ...